src/tsickle.ts

```diff
@@ -2017,6 +2025,41 @@ export function emitWithTsickle(
   };
 }
 
+function addClutzAliases(
+    dtsFileContent: string, sourceFiles: ts.SourceFile[], typeChecker: ts.TypeChecker,
+    host: TsickleHost): string {
+  let reexports = '';
+  for (const sf of sourceFiles!) {
+    const moduleSymbol = typeChecker.getSymbolAtLocation(sf);
+    const moduleExports = moduleSymbol && moduleSymbol.exports || new Map<string, ts.Symbol>();
+
+    for (const e of moduleExports.entries()) {
+      const symbolName = e[0];
+      const symbol = e[1];
+      const clutzSymbolName = 'module$contents$' +
+          host.pathToModuleName('', sf.fileName).replace(/\./g, '$') + '_' + e[0];
+      const symbolType = typeChecker.getTypeAtLocation(symbol.valueDeclaration!);
+      if ((symbolType as ts.ObjectType).objectFlags &&
```

Comment:
can we have a safe way to handle non-object types, instead of assertions. I.e. just bail if non-object type. I think you can get those cases if one writes CommonJs style typescript, instead of ES6
 
```
export = 0;
```